Header header
ticker_publisher/ticker tick
float64 short_emaf
float64 long_emaf
float64 macd
float64 signal
float64 macd_hist